tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
Also `esModuleInterop: true`

Reply:
Don't think it does anything because compilation happens through Babel not `tsc`, but I set it anyway.